libcxx/src/locale.cpp

```diff
@@ -34,7 +34,8 @@
 #  define _CTYPE_DISABLE_MACROS
 #endif
 
-#if !defined(_LIBCPP_MSVCRT) && !defined(__MINGW32__) && !defined(__BIONIC__) && !defined(__NuttX__)
+#if !defined(_LIBCPP_MSVCRT) && !defined(__MINGW32__) && !defined(__BIONIC__) && !defined(__NuttX__) ||                \
+    defined(__LLVM_LIBC__)
 #  include <langinfo.h>
 #endif
```

Comment:
`#if __has_include`?

Reply:
I was following what the other targets do, but I could replace it with a has include if needed.

Reply:
Updated, guess we'll see if this passes CI.